libavcodec/x86/vvcdsp_init.c

```diff
@@ -120,19 +120,17 @@ static void alf_filter_chroma_8_avx2(uint8_t *dst, ptrdiff_t dst_stride, const u
         c->alf.filter[CHROMA] = alf_filter_chroma_##depth##_avx2;               \
     } while (0)
 
-#define SAO_BAND_FILTER_FUNCS(bitd, opt)                                                                                      \
```

Comment:
We do not need to change this. how about we rebase and combine this with other commit